scripts/core/editor3/actions/spellchecker.jsx

```diff
@@ -0,0 +1,13 @@
+export function replaceWord(word) {
+    return {
+        type: 'REPLACE_WORD',
+        payload: word
+    };
+}
+
+export function showContextMenu(contextMenuData) {
```

Comment:
All over the application I used the name contextMenuData and I think it is better to keep that name in order to don't create confusion to the reader. I agree that local variable should be short but for parameters I think that a more explanatory name will help. This variable is used only once so I think it don't creates any complication or reading difficulty.